modules/generating-icsp-object-scoped-to-a-registry.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc:
+
+[id="generating-icsp-object-scoped-to-a-registry_{context}"]
+= Widening the mirror registry scope of the ImageContentSourcePolicy object
+
+Use the `--icsp-scope=registry` flag with the `oc adm catalog mirror` command to widen the mirror registry scope in the `ImageContentSourcePolicy` object. This minimizes the number of times the cluster nodes reboot when updating a registry entry.
+
+The `oc adm catalog mirror` command generates an `ImageContentSourcePolicy` resource that maps the original container image repository to a local image catalog repository that you create and mirrors it. This typically occurs inside a disconnected environment. When you apply a new or modified `ImageContentSourcePolicy` resource to a cluster, the `oc adm catalog mirror` command generates a configuration file and places it onto each node. This configuration file is used by CRI-O, the container runtime that manages the containers running on the nodes. When you update the configuration file for a node, the node reboots. A widely scoped `ImageContentSourcePolicy` resource reduces the number of times the cluster nodes need to reboot in response to changes to the resource.
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in as a user with `cluster-admin` privileges.
+* Configure a local image catalog repository for use in your disconnected cluster. See xref:../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for more details.
+
+.Procedure
+
+To widen the mirror registry scope in the `ImageContentSourcePolicy` resource, do the following:
+
+. Run the following command, specifying values for `<local_registery>` and `<pull_secret_file>`:
++
+[source,terminal]
+----
+$ oc adm catalog mirror <local_registery>/olm/redhat-operators:v1 <local_registery> -a <pull_secret_file> --icsp-scope=registry
+----
++
+<1> `<local_registery>` is the local registry you have configured for your disconnected cluster, for example `local.registry:5000`.
+<2> `<pull_secret_file>` is the `registry.redhat.io` pull secret in `.json` file format downloaded from the link:https://cloud.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {cloud-redhat-com} site.
++
+.Expected result
+The `oc adm catalog mirror` command creates a `/redhat-operator-index-manifests` directory and generates `imageContentSourcePolicy.yaml` and `mapping.txt` files.
+
+. Apply the new `ImageContentSourcePolicy` resource to the cluster:
++
+[source,terminal]
+----
+$ oc apply -f redhat-operator-index-manifests/imageContentSourcePolicy.yaml
+----
+
+.Verification steps
+
+* Verify that `oc apply` successfully applies the change to `ImageContentSourcePolicy`:
```

Comment:
I'm usually a present-tense fanatic, but past-tense "applied" seems reasonable because this is definitely sequential.  Up to you.